projects/plugins/jetpack/_inc/lib/class-jetpack-podcast-helper.php

```diff
@@ -16,13 +16,48 @@ class Jetpack_Podcast_Helper {
 	 */
 	protected $feed = null;
 
+	/**
+	 * The number of seconds to cache the podcast feed data.
+	 * This value defaults to null, so we don't override the core filters for RSS feeds in WordPress.
+	 * The value can be overridden specifically for podcasts using the
+	 * `jetpack_podcast_feed_cache_timeout` filter. Note that the cache timeout value
+	 * for all RSS feeds can be modified using the `wp_feed_cache_transient_lifetime`
+	 * filter from WordPress core.
+	 *
+	 * @see https://developer.wordpress.org/reference/hooks/wp_feed_cache_transient_lifetime/
+	 * @see WP_Feed_Cache_Transient
+	 *
+	 * @var int|null
+	 */
+	protected $cache_timeout = null;
+
 	/**
 	 * Initialize class.
 	 *
 	 * @param string $feed The RSS feed of the podcast.
 	 */
 	public function __construct( $feed ) {
 		$this->feed = esc_url_raw( $feed );
+
+		/**
+		 * Filter the number of seconds to cache a specific podcast URL for. The returned value will be ignored if it is null or not a valid integer.
+		 * Note that this timeout will only work if the site is using the default `WP_Feed_Cache_Transient` cache implementation for RSS feeds,
+		 * or their cache implementation relies on the `wp_feed_cache_transient_lifetime` filter.
+		 *
+		 * @since $$next-version$$
+		 * @see https://developer.wordpress.org/reference/hooks/wp_feed_cache_transient_lifetime/
+		 *
+		 * @param int|null $cache_timeout The number of seconds to cache the podcast data. Default value is null, so we don't override any defaults from existing filters.
+		 * @param string   $podcast_url   The URL of the podcast feed.
+		 */
+		$podcast_cache_timeout = apply_filters( 'jetpack_podcast_feed_cache_timeout', $this->cache_timeout, $this->feed );
+
+		// Make sure we force $this->cache_timeout to be int|null.
+		if ( $podcast_cache_timeout === null || ! is_int( $podcast_cache_timeout ) ) {
+			$this->cache_timeout = null;
+		} else {
+			$this->cache_timeout = (int) $podcast_cache_timeout;
+		}
```

Comment:
`$this->cache_timeout` is already null by default, so you could just do:

```suggestion
		if ( is_int( $podcast_cache_timeout ) ) {
			$this->cache_timeout = $podcast_cache_timeout;
		}
```

Reply:
Great point! I've simplified this in 085ead2, though I have kept the cast to `int`.